tests/runtime/ray/test_integration.py

```diff
@@ -263,12 +265,60 @@ def test_status_after_awaiting(self, runtime: _dag.RayRuntime, trial):
                 # during start and finish of a task. Thus it is >=, not >
                 assert (status.end_time - status.start_time).total_seconds() >= 0
 
+        @freeze_time("9999-01-01")
+        def test_normalizes_end_times(self, runtime: _dag.RayRuntime):
+            # Given
+            wf_def = _example_wfs.serial_wf_with_slow_middle_task.model
+            run_id = runtime.create_workflow_run(wf_def, None, False)
+            runtime.stop_workflow_run(run_id)
+
+            # Block until wf completes
+            _wait_to_finish_wf(run_id, runtime)
+
+            # When
+            run = runtime.get_workflow_run_status(run_id)
+
+            # Then
+            assert run.id == run_id
+            assert (
+                run.status.state == State.TERMINATED
+            ), f"Invalid state. Full status: {run.status}. Task runs: {run.task_runs}"
+            assert run.status.start_time is not None
+            assert run.status.end_time is not None
+            assert run.workflow_def == wf_def
+
+            # The added end times should be 'now'
+            assert run.status.end_time == datetime(9999, 1, 1, tzinfo=timezone.utc)
+            # freeze_time doesn't affect the ray process, so the start times
+            # are real and therefore earlier than the end time (if this is
+            # untrue, hello from the distant past).
+            assert run.status.start_time < run.status.end_time
+
+            for task_run in run.task_runs:
+                assert task_run.invocation_id in wf_def.task_invocations.keys()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏What's this assertion for?

Reply:
Good catch, accidentally replicated from the test above. Removed in [26a0e43](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/329/commits/26a0e432228da323e89a30e2f6cca2a861d575d0)